README.md

```diff
@@ -0,0 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Book 
```

Comment:
- [ ] Kindly delete the commented code here

  ![udeagha-comm](https://user-images.githubusercontent.com/36542986/171028439-1ebb16a7-65fc-4235-9275-58d577e53614.jpg)


- [ ] Your data in the `localStorage` should be presented on the page after reload. Your data is rightly stored in the storage but does not reflect on the page when reloaded. Kindly fix that.

  Books are added | Page is reloaded
  :---:|:---:
  ![udeagha-1](https://user-images.githubusercontent.com/36542986/171028883-88ecb520-f42b-435d-bb30-9b47e01d6386.jpg) | ![udeagha-2](https://user-images.githubusercontent.com/36542986/171028886-60d621d8-758a-48dd-8351-a2a5efa8e5ee.jpg)